src/Admin/Vue.php

```diff
@@ -27,6 +27,9 @@ public function init() {
 			$logged = $this->payplug_section_logged();
 			$payplug_wooc_settings = get_option( 'woocommerce_payplug_settings', [] );
 			unset($payplug_wooc_settings["payplug_live_key"]);
+			unset($payplug_wooc_settings["payplug_test_key"]);
```

Comment:
we need the test key for inactive merchants

Reply:
Same thing as merchant_id, payplug_test_key is not used in vue side, only used in server side